.nvmrc

```diff
@@ -0,0 +1 @@
+16.14.0
```

Comment:
Hm, not sure. I always try to keep these things/configs limited. 
But if it's a hard requirement, it might make sense to add a note in the readme/docs

Reply:
Hey wslyvh you are right is not necessary 💪